File: DevExpress-Demo/CS/ControlsDemo.Wpf/Services/TileBarService.cs
using System;
using System.Linq;
using DevExpress.Mvvm.UI;
using DevExpress.Xpf.Navigation;

namespace ControlsDemo {
    public interface ITileBarService {
        void CloseFlyout();
    }
    public class TileBarService : ServiceBase, ITileBarService {
        public void CloseFlyout() {
            TileBar tileBar = AssociatedObject as TileBar;
            if(tileBar != null)
                tileBar.CloseFlyout();
        }
    }
}

File: DevExpress-Demo/VB/ControlsDemo.Wpf/Services/TileBarService.vb
Imports DevExpress.Mvvm.UI
Imports DevExpress.Xpf.Navigation

Namespace ControlsDemo

    Public Interface ITileBarService

        Sub CloseFlyout()

    End Interface

    Public Class TileBarService
        Inherits ServiceBase
        Implements ITileBarService

        Public Sub CloseFlyout() Implements ITileBarService.CloseFlyout
            Dim tileBar As TileBar = TryCast(AssociatedObject, TileBar)
            If tileBar IsNot Nothing Then tileBar.CloseFlyout()
        End Sub
    End Class
End Namespace
